avs/vis_avs/e_superscope.cpp

```diff
@@ -111,9 +111,17 @@ int E_SuperScope::render(char visdata[2][2][576],
     int sign_invert = this->config.audio_source ? 0 : 128;
 
     if (this->config.audio_channel == 0) {
-        for (x = 0; x < 576; x++)
-            center_channel[x] = visdata[1 - this->config.audio_source][0][x] / 2
-                                + visdata[1 - this->config.audio_source][1][x] / 2;
+        if (this->config.audio_source == 0) {
+            for (x = 0; x < 576; x++)
```

Comment:
Please add braces here -- I know they're not present in the original (an oversight on my part), but while we're touching it, might as well add them.

Reply:
@grandchild alright, fixed and pushed